### DISCUSSION TITLE
Increased vehicle spawning in towns

### DISCUSSION THREAD

**Discussion by IkanTerbakor** [Vdauphin/HeartsAndMinds#1659] [Question and answer]
Hello there!

How can i increase the spawn rate of vehicles(on ground and air) in towns?
Thanks for the mission <3

Cheers!

**Comment by Vdauphin** [comment]
Hello,

City doesn't spawn vehicle
Vehicles in H&M are generated by the patrol system:  https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L230-L235

There is an option to spawn vehicles in side mission or cache: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L224-L229

Cheers

**Reply by IkanTerbakor** [reply]
can i increase the number of patrols past 10?

**Reply by Vdauphin** [reply]
Yes

**Reply by IkanTerbakor** [reply]
Thank you
